docs/source/install/Users.rst

```diff
@@ -0,0 +1,92 @@
+
+A Detailed Walk Through
+#######################
+
+First, save the following lines as :code:`ghz_3.py` in a folder where you want to install QMAP and run the example:
+
+    .. code-block:: python
+
+        from qiskit import QuantumCircuit
+        from qiskit.providers.fake_provider import FakeLondon
+        from mqt import qmap
+
+        # create your quantum circuit
+        circ = QuantumCircuit(3)
+        circ.h(0)
+        circ.cx(0, 1)
+        circ.cx(0, 2)
+        circ.measure_all()
+        print(circ.draw(fold=-1))
+
+        # compile circuit to 5-qubit London Architecture
+        circ_mapped, results = qmap.compile(circ, FakeLondon())
+        print(circ_mapped.draw(fold=-1))
+
+        # print the result
+        print("Additional Gates:%d" % (results.json()["statistics"]["additional_gates"]))
+
+Then, the following snippet shows the installation process from setting up the virtual environment to running a small example program.
+
+    .. code-block:: console
+
+        $ python3 -m venv venv
+        $ . venv/bin/activate
+        (venv) $ pip install -U pip setuptools wheel
+        (venv) $ pip install mqt.qmap
+        (venv) $ python3 ghz_3.py
+                ┌───┐           ░ ┌─┐
+           q_0: ┤ H ├──■────■───░─┤M├──────
+                └───┘┌─┴─┐  │   ░ └╥┘┌─┐
+           q_1: ─────┤ X ├──┼───░──╫─┤M├───
+                     └───┘┌─┴─┐ ░  ║ └╥┘┌─┐
+           q_2: ──────────┤ X ├─░──╫──╫─┤M├
+                          └───┘ ░  ║  ║ └╥┘
+        meas: 3/═══════════════════╩══╩══╩═
+                                   0  1  2
+                                                            ░
+                    ┌───┐┌───┐          ┌─┐
+           q_0 -> 0 ┤ H ├┤ X ├──■───────┤M├──────
+                    └───┘└─┬─┘┌─┴─┐     └╥┘┌─┐
+           q_1 -> 1 ───────■──┤ X ├──■───╫─┤M├───
+                              └───┘┌─┴─┐ ║ └╥┘┌─┐
+           q_2 -> 2 ───────────────┤ X ├─╫──╫─┤M├
+                                   └───┘ ║  ║ └╥┘
+           q_3 -> 3 ─────────────────────╫──╫──╫─
+                                         ║  ║  ║
+           q_4 -> 4 ─────────────────────╫──╫──╫─
+                                         ║  ║  ║
+               c: 3/═════════════════════╩══╩══╩═
+                                         1  0  2
+
+        Additional Gates:1
+
+
+Building from Source for Performance
+####################################
+
+In order to get the best performance out of QMAP and enable platform-specific compiler optimizations that cannot be enabled on portable wheels, it is recommended to build the package from source via:
+
+    .. code-block:: console
+
+        (venv) $ pip install mqt.qmap --no-binary mqt.qmap
+
+This requires a `C++ compiler <https://en.wikipedia.org/wiki/List_of_compilers#C++_compilers>`_ compiler supporting *C++17* and a minimum `CMake <https://cmake.org/>`_ version of *3.14*.
```

Comment:
This is also missing the Z3 requirement. Again, this can be copied/adapted from the README.

Reply:
Done.